src/preferencesdialog.ui

```diff
@@ -201,6 +201,13 @@
          </property>
         </widget>
        </item>
+       <item row="7" column="0">
+        <widget class="QCheckBox" name="smoothOnZoomBox">
+         <property name="text">
+          <string>Smoothing images on zoom</string>
```

Comment:
Please just change this to `Smooth images on zooming`. If it's changed after the merge, translations might be removed.

Reply:
oops, sorry, done